lib/setting.rb

```diff
@@ -88,7 +82,7 @@ def self.available_settings
   #=================================================================
 
   def initialize
-    @available_settings ||= SettingHash.new
+    @available_settings ||= Hashie::Mash.quiet(:default).new
```

Comment:
@zackse should Fittings require Hashie `>= 4.0` since that was when the `Mash.quiet` was introduced?

Reply:
Thanks, posted #32 for this change.